Scenarios/AKS-Secure-Baseline-PrivateCluster/Apps/SimpleGMSAApp/Getting-Started.md

```diff
@@ -62,40 +65,78 @@ Follow the steps [here](https://learn.microsoft.com/virtualization/windowscontai
 1. If you are running these commands on your domain controller that is a Windows Server machine, you may have trouble
 installing the [Kubernetes CLI (kubectl)](https://kubernetes.io/docs/tasks/tools/install-kubectl-windows/) and [kubelogin](https://github.com/Azure/kubelogin). The error would mention "Install-AzAksKubectl". If so, you will need to manually install both following the links above.
 
-## How to Validate Your GMSA Integration
+## How to Validate Your GMSA Integration before Deployment
+
+1. To validate that your cluster is successfully retrieving your GMSA, go into your domain controller local server menu, go to Tools and select Event Viewer. Look under ActiveDirectory events. Look at the contents of the most recent events for a message that says "A caller successfully fetched the password of a group managed service account." The IP address of the caller should match one of your AKS cluster IPs.
+
+## Build Simple Application Container Image
+
+The Dockerfile for the Simple application is in the repository you've already cloned to your jumpbox in the previous step. 
+
+Navigate to the [Simple Application Dockerfile](../SimpleGMSAApp/application/SimpleApp.Dockerfile), build and tag the containers with the name of your Azure Container Registry and push the images to ACR. Use the ACR you created as apart of the reference implementation deployment. // Make sure it is the correct ACR
+
+```PowerShell
+# enter the name of your ACR below
+$SPOKERG=<resource group name for spoke>
+$ACRNAME=$(az acr show --name <ACR NAME> --resource-group $SPOKERG --query "name" --output tsv)
+
+cd aks-baseline-windows/Scenarios/AKS-Secure-Baseline-PrivateCluster/Apps/SimpleGMSAAPP/application
+
+docker build -t $ACRNAME.azurecr.io/simpleapp:v1 -f SimpleApp.Dockerfile .
```

Comment:
definitely took this all further than I was expecting :)  I assumed you'd simply `az acr import mcr.microsoft.com/windows/servercore/iis:windowsservercore-ltsc2019` and then make sure the **deployment_sampleapp.yml** file just pulled from your ACR instance.  Is it expected for the jumpbox to actually have docker installed?  Feels like this might be over complicating it.  Thoughts?

Reply:
Yeah that would've been much simpler haha. I think for a simple app like this it makes sense to not require Docker, but when we get into legacy, we need to do multistage builds to optimize the images. For this app, I'll use acr import and for legacy, I'll update the instructions to use az acr build